dev-docs/AUTOMATE_DEV_ENV.md

```diff
@@ -0,0 +1,43 @@
+The goal of this document is to set up a development environment of 
+automate for Chef Server related work. If you encounter a process
+or feature that isn’t straightforward or took you time to understand,
+consider writing an explanation here.
+
+## Setup
```

Comment:
Duplicating the Automate dev docs is not a recommended practice. Your local docs **will** get out of sync with changes in the main repo. The best practice is to link to the main docs and only note those things that are different for this repository

https://github.com/chef/automate/blob/master/dev-docs/DEV_ENVIRONMENT.md

Reply:
Thanks for the inputs! I have made the changes for the same.